modules/Bio/EnsEMBL/Variation/Pipeline/VariationConsequence_conf.pm

```diff
@@ -63,16 +63,16 @@ sub default_options {
         hive_no_init => 0,
         # the location of your checkout of the ensembl API (the hive looks for SQL files here)
         
-        ensembl_cvs_root_dir    => $ENV{'HOME'} . '/src',
-        hive_root_dir           => $ENV{'HOME'} . '/src/ensembl-hive', 
+        ensembl_cvs_root_dir    => '/hps/software/users/ensembl/repositories/'. $login . '/src',
+        hive_root_dir           => '/hps/software/users/ensembl/repositories/'. $login . '/src/ensembl-hive', 
```

Comment:
I would prefer to not have the location in the pipeline config and suggest to replace L66 and L67 with:
ensembl_cvs_root_dir    => $ENV{'ENSEMBL_ROOT_DIR'} || $self->o('ensembl_cvs_root_dir'),
hive_root_dir           => $self->o('ensembl_cvs_root_dir') . '/ensembl-hive',